config/routes.rb

```diff
@@ -36,4 +36,6 @@
       resource :users, only: :show, path: :me
     end
   end
+
+  get '/openapi', to: 'openapi#show'
```

Comment:
How about moving it to the namespace `api`?

```ruby
namespace :api do
  namespace :v1 do
    #...
  end

  namespace :docs do
     get '/openapi', to: 'openapi#show'
  end
end
```

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/89/commits/6391ef47b371a40433b500dd5c0775c31a735eca